README.md

```diff
@@ -24,16 +25,19 @@ Share text using a native sharing dialog.
 **PARAMETERS**
 * ```text``` (string) - The text to share
 
-### share.image(bytes, [text])
-Share an image (with optional text) using a native sharing dialog. The image format must be supported by ```BitmapFactory.decodeByteArray()``` on Android and ```UIImage.initWithData``` on iOS/OSX. On Android the image will first be stored locally as a file and then shared using a FileProvider. In order for this to work you need to follow the additional setup steps for Android mentioned above.
+### share.image(bytes, [text], [file_name])
+Share an image (with optional text) using a native sharing dialog. The image format must be supported by ```BitmapFactory.decodeByteArray()``` on Android, ```UIImage.initWithData``` on iOS/OSX and [DataURI](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) on HTML5. On Android the image will first be stored locally as a file and then shared using a FileProvider. In order for this to work you need to follow the additional setup steps for Android mentioned above.
 
 **PARAMETERS**
 * ``` bytes``` (string) - The image bytes to share
 * ```text``` (string) - Optional text to share
+* ```file_name``` (string) - Optional Only for `HTML5` image name. Default `"file.png"`
 
-### share.file(path, [text])
+### share.file(path, [text], [file_name])
```

Comment:
I'm thinking about argument order and how to make HTML5 more closely resemble  the other platforms. With your suggestion both path and text are given a different meaning in html5

Wouldn't this be better?

```
-- html5
-- third argument is a table with options (can be used for extra params on all platforms)
share.file("file.txt", "file contents", { type = "text/plain" }

-- others
share.file("path/to/file.txt", "extra text")
```

Reply:
@britzl I agree. Changed parameters for html version